test/integration/model/geometry.test.js

```diff
@@ -32,7 +36,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         await Pub.sync({ force: true });
         const pub = await Pub.create({ location: point });
         expect(pub).not.to.be.null;
-        expect(pub.location).to.be.deep.eql(point);
+        expect(pub.location).to.deep.include(point);
```

Comment:
this shouldn't have changed. There shouldn't be additional properties

It also seems very unrelated to this PR. Can you revert all changes to this file and, if needed, move to another PR?